ml_datasets/loaders/mnist.py

```diff
@@ -97,3 +124,21 @@ def load_kuzushiji_mnist(
     test_images = np.load(test_img_path)['arr_0']
     test_labels = np.load(test_label_path)['arr_0']
     return (train_images, train_labels), (test_images, test_labels)
+
+
+def load_emnist(path="matlab.zip", split='digits'):
```

Comment:
`"matlab.zip"` is hardcoded twice in this file - perhaps it should be stored in a separate variable so this code wouldn't unexpectedly break if we'd update `EMNIST_URL` sometime in the future?